airbyte-cdk/bulk/core/base/src/test/kotlin/io/airbyte/cdk/initialization/TestApplicationYaml.kt

```diff
@@ -0,0 +1,29 @@
+package io.airbyte.cdk.initialization
+
+import io.micronaut.context.annotation.Bean
+import io.micronaut.context.annotation.Factory
+import io.micronaut.context.annotation.Value
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest
+import jakarta.inject.Inject
+import org.junit.jupiter.api.Test
+
+
+@MicronautTest
+class TestApplicationYaml {
+    @Inject lateinit var testBean: TestBean
+
+    @Test
+    fun testApplicationYamlInjection() {
+        assert(testBean.testApplicationYamlEntry == "value")
+    }
+}
+
+data class TestBean(val testApplicationYamlEntry: String)
+
+@Factory
+class TestFactory {
+    @Bean
+    fun testBean(@Value("\${airbyte.test}") testValue: String): TestBean {
```

Comment:
Is it also worth adding tests to validate those default values in the real application.yml?  Seems like something we should add here just to validate the defaults.

Reply:
I have updated the test to check that we are properly getting the value based on an env var. I can't test the default value because we are using the test application.yaml

Reply:
Spoke through a VC, done.